Gem/Settings/ViewModels/CurrencySceneViewModel.swift

```diff
@@ -66,6 +70,70 @@ class CurrencySceneViewModel: ObservableObject {
 
 extension Locale.Currency {
     var title: String {
-        return String(format: "%@ - %@", identifier, Locale.current.localizedString(forCurrencyCode: identifier) ?? .empty)
+        let localizedCurrencyId = Locale.current.localizedString(forCurrencyCode: identifier) ?? .empty
+        let currencyIdenifier = identifier
+
+        if let emojiFlag = emojiFlag {
+            return "\(emojiFlag) \(currencyIdenifier) - \(localizedCurrencyId)"
+        } else {
+            return "\(currencyIdenifier) - \(localizedCurrencyId)"
+        }
+    }
+
+    private var emojiFlag: String? {
+        return Self.flag(for: identifier)
+    }
+
+    fileprivate static func flag(for identifier: String) -> String? {
+        return Self.emojiFlags[identifier] ?? ""
```

Comment:
if you return "String?` should we avoid returning `?? ""`

Reply:
correct, typo